README.md

```diff
@@ -1,7 +1,23 @@
-If you want to work on this repo, you can preview the site as you work.
+##	Documentation & User Guides
 
-Linux Mint / Ubuntu instructions:
+This repo is the source for the official [Pi-hole documentation](https://docs.pi-hole.net/).
+
+#### How to contribute.
+To add a new link on the navigation panel you need to edit the `mkdocs.yml` file in the root of the repo. There is a guide for building the navbar [on the mkdocs wiki]( https://www.mkdocs.org/user-guide/configuration/#nav)
+
+To add a new document or guide. 
+
+- Navigate to the directory where it will be hosted.  
+	EG. guides are in `REPO/docs/guides`
+- Create the file using a URL friendly filename.  
+	EG. `docs/guides/url-friendly.md`
+- Edit your document using Markdown, there are loads of resources available for the correct syntax.
 
+
+#### Testing your changes.
+Whilst working on this repo, it is advised that you review your own changes locally before commiting them. This can be done by using the `mkdocs serve` command. 
+
+Linux Mint / Ubuntu instructions:
```

Comment:
These commands are the same for most platforms, so this distro disclaimer can be removed.